apps/web/ee/components/support/HelpMenuItem.tsx

```diff
@@ -1,13 +1,213 @@
+  const { t } = useLocale();
+
+  const onRatingClick = (value: string) => {
+    setRating(value);
+    setDisableSubmit(false);
+  };
+
+  const sendFeedback = async (rating: string, comment: string) => {
+    setLoading(true);
+    try {
+      const body = {
+        rating: rating,
+        comment: comment,
+      };
+
+      const res = await fetch("api/send-feedback", {
+        method: "POST",
+        body: JSON.stringify(body),
+        headers: {
+          "Content-Type": "application/json",
+        },
+      });
+
+      if (res.ok) {
+        setSuccessMessage(true);
+        setDisableSubmit(true);
+      } else {
+        setErrorMessage(true);
+      }
+    } catch (error) {
+      setErrorMessage(true);
+    }
+    setLoading(false);
+  };
+
   return (
-    <>
-      <IntercomMenuItem />
-      <ZendeskMenuItem />
-      <HelpscoutMenuItem />
-    </>
+    <div className="w-full border-gray-300 bg-white shadow-sm md:w-[150%]">
+      <div className=" w-full p-5">
+        <p className="mb-1 text-neutral-500">{t("resources").toUpperCase()}</p>
+        <a
+          href="https://docs.cal.com/"
+          target="_blank"
+          className="flex w-full py-2 pr-4 text-sm font-medium text-gray-700 hover:bg-gray-100 hover:text-gray-900"
+          rel="noreferrer">
+          {t("support_documentation")}
+          <ExternalLinkIcon
+            className={classNames(
+              "text-neutral-400 group-hover:text-neutral-500",
+              "ml-1 h-5 w-5 flex-shrink-0 ltr:mr-3"
+            )}
+          />
+        </a>
+        <a
+          href="https://developer.cal.com/"
+          target="_blank"
+          className="flex w-full py-2 pr-4 text-sm font-medium text-gray-700 hover:bg-gray-100 hover:text-gray-900"
+          rel="noreferrer">
+          {t("developer_documentation")}
+          <ExternalLinkIcon
+            className={classNames(
+              "text-neutral-400 group-hover:text-neutral-500",
+              "ml-1 h-5 w-5 flex-shrink-0 ltr:mr-3"
+            )}
+          />
+        </a>
+        <ContactMenuItem />
+      </div>
+
+      <hr className=" bg-gray-200" />
+      <div className="w-full p-5">
+        <p className="mb-1 text-neutral-500">{t("feedback").toUpperCase()}</p>
+        <p className="flex w-full py-2 text-sm font-medium text-gray-700">{t("comments")}</p>
+        <textarea
+          id="comment"
+          name="comment"
+          rows={3}
+          onChange={(event) => setComment(event.target.value)}
+          className="my-1 block  w-full rounded-sm border-gray-300 py-2 pb-2 shadow-sm sm:text-sm"></textarea>
+
+        <div className="my-3 flex justify-end">
+          <button
+            className={classNames(
+              "m-1 items-center justify-center p-1.5 outline outline-1 outline-gray-300 hover:bg-gray-100",
+              rating === "Extremely unsatisfied" ? "bg-gray-300" : ""
+            )}
+            onClick={() => onRatingClick("Extremely unsatisfied")}>
```

Comment:
@PeerRich @Jaibles @alannnc changed the ratings to be strings 
- Extremely unsatisfied
- Unsatisfied
- Satisfied
- Extremely satisfied